is4_fib.loboc

```diff
@@ -0,0 +1,30 @@
+// ISEQ4: Calculates the nth fibinacci number
+//tlc type: run
+//tlc case: 1=0
+//tlc case: 1=1
+//tlc case: 2=2
+//tlc case: 3=3
+//tlc case: 5=4
+//tlc case: 8=5
+//tlc case: 13=6
+//tlc case: 21=7
+
+unsigned n;
+
+{
+    unsigned[n] fib_nums;
```

Comment:
Hmmm, I'm not sure this was ever hammered down in a spec, but my compiler won't allow non-constant size arrays.